recipes/rmlui/4.x/test_v1_package/example.cpp

```diff
@@ -0,0 +1,28 @@
+#include <RmlUi/Core.h>
```

Comment:
Is this file a duplicate? I think the one in `test_package` is being used, right?

Reply:
woops yeah, that's right. Will remove